Tests/XCUITests/SearchTest.swift

```diff
@@ -226,6 +227,10 @@ class SearchTests: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: "google")
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/1614032
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/966835
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/77246
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/1613988
```

Comment:
This is one of the things we want to avoid, having more than one manual tests refering to one automated tests, there should be a one to one correspondence.